amclient/amclientargs.py

```diff
@@ -150,6 +157,20 @@
 JOB_NAME = Opt(
     name="job-name", metavar="JOBNAME", help="Filter by name", default=None, type=None
 )
+LOCATION_DESCRIPTION = Opt(
+    name="location-description",
+    metavar="LOCATIONDESCRIPTION",
+    help="Description for the storage location",
+    default=None,
+    type=None,
+)
+DEFAULT = Opt(
```

Comment:
Is this option named so generic in order to be reused in future commands?

Reply:
Yeah, I figured there might be other commands that offer the option of having it toggled on as default so thought I'd leave it like this. I feel otherwise there's quite an overhead to adding args in this tool the way it is written :\